### ISSUE TITLE
saving trained tensor flow data on google colab

### ISSUE THREAD

**Issue by power9799** [tensorflow/tensorflow#36460]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): stock example
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Mac catalina
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary):
- TensorFlow version (use command below):
- Python version: python 3.0 on colab
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: none
- GPU model and memory: colab

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with: 1. TF 1.0: `python -c "import
tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"` 2. TF 2.0: `python -c
"import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**
I ran a walkthrough using python notebook 3 on google colab and every time I run the model, I have to train with the data and then run the prediction. but I'm curious is there anyway to save your data on google colab, so I don't have to  train the model every time when I need to do the prediction.
**Describe the expected behavior**
I want it to save my trained data.
**Code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by gadagashwini** [comment]
@power9799, Save your model and load when ever you want to evaluate or predict. Save the model in .h5 or .pb format in your google drive. Please take a look at the [guide](https://www.tensorflow.org/tutorials/keras/save_and_load) to Save and load model in Tensorflow. Thanks!

**Comment by power9799** [comment]
oh 

> @power9799, Save your model and load when ever you want to evaluate or predict. Save the model in .h5 or .pb format in your google drive. Please take a look at the [guide](https://www.tensorflow.org/tutorials/keras/save_and_load) to Save and load model in Tensorflow. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by gadagashwini** [comment]
@power9799, Closing this issue since it is resolved. Please feel free to open if still issue persists. Thanks
